actions/admin/helpers.go

```diff
@@ -0,0 +1,76 @@
+package admin
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/bitcoin-sv/spv-wallet/actions/common"
+	"github.com/bitcoin-sv/spv-wallet/engine"
+	"github.com/bitcoin-sv/spv-wallet/engine/datastore"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/mappings"
+	"github.com/bitcoin-sv/spv-wallet/models/filter"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+	"github.com/bitcoin-sv/spv-wallet/server/reqctx"
+	"github.com/gin-gonic/gin"
+	"github.com/rs/zerolog"
+)
+
+// Helper function to encapsulate error response logic
+func handleError(c *gin.Context, err error, logger *zerolog.Logger) {
+	spverrors.ErrorResponse(c, err, logger)
+}
```

Comment:
To me it's unnecessary function which adds complexity with no benefit.
I would use `spverrors.ErrorResponse` "in-place" as we usually do.